src/http/client.rs

```diff
@@ -4416,16 +4414,16 @@ impl Http {
     ///
     /// This is a function that performs a light amount of work and returns an empty tuple, so it's
     /// called "self.wind" to denote that it's lightweight.
```

Comment:
```suggestion
    /// Performs a request and, if successful, verifies that the response
    /// status code is equal to the `NO_CONTENT` status code. 
    /// 
    ///
    /// This is a function that performs a light amount of work and returns an empty tuple, so it's
    /// called "self.wind" to denote that it's lightweight.
```

This is meant to include the two lines above, but Github isn't allowing me to select them.

Reply:
```rust
    /// Performs a request and verifies that Discord responds with [`StatusCode::NO_CONTENT`].
    ///
    /// This is a function that performs a light amount of work and returns the unit type, so it's
    /// called "self.wind" to denote that it's lightweight.
```
Is this good for you?

Reply:
Yeah, that's fine.